Week1/solution 4.py

```diff
@@ -0,0 +1,20 @@
+# Python program to check if the number is an Armstrong number or not
+
+# take input from the user
+num = int(input("Enter a number: "))
+
+# initialize sum
+sum = 0

```

Comment:
Why You initialised sum value to 0, what if there were no initial value provided ?

Reply:
A default value of 0 ensures the expected behavior of returning the total sum of the input values.